frontend/src/views/ElementForm/index.vue

```diff
@@ -24,9 +24,15 @@
         </DsfrTabContent>
 
         <DsfrTabContent panel-id="history-content" tab-id="history">
-          <div class="-my-8">
+          <div class="-my-8 mb-4">
             <DsfrTable :headers="headers" :rows="historyDataDedup" />
           </div>
+          <p v-if="element && element.originDeclaration">
+            <router-link :to="{ name: 'InstructionPage', params: { declarationId: element.originDeclaration } }">
+              La déclaration qui a demandé cet ingrédient
```

Comment:
Si ce n'est pas trop long je dirais p-e :

```suggestion
              Déclaration ayant demandé l'ajout de cet ingrédient
```